### PR TITLE
fix(general-petitions): pagination limit (temp) and singEndorsement qury fix

### PR DESCRIPTION
**Pull request by albinagu**
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/general-petition/src/hooks/useHasEndorsed.ts [hunk 1]
```diff
@@ -18,5 +18,7 @@ export const useHasEndorsed = (endorsementListId: string) => {
       },
     },
   )
-  return endorsement?.endorsementSystemUserEndorsements?.hasEndorsed
+
+  console.log(endorsement)
```
Comment (jGeir) [inline libs/application/templates/general-petition/src/hooks/useHasEndorsed.ts:22]: is this console log required for future development?

**Review by jGeir — APPROVED** [review]
lgtm

File: libs/application/templates/general-petition/src/hooks/useHasEndorsed.ts (hunk 1 above)
Comment (alexandradilja) [reply libs/application/templates/general-petition/src/hooks/useHasEndorsed.ts:22]: Console.log slipped through?

**Review by alexandradilja — APPROVED** [review]
One comment, otherwise lgtm

**Pull request merged**